[2024-11-03T18:59:28.129+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Data_Preprocessing.encode_data_task manual__2024-11-03T18:59:05.994397+00:00 [queued]>
[2024-11-03T18:59:28.146+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_Data_Preprocessing.encode_data_task manual__2024-11-03T18:59:05.994397+00:00 [queued]>
[2024-11-03T18:59:28.147+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T18:59:28.147+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-11-03T18:59:28.148+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-03T18:59:28.171+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): encode_data_task> on 2024-11-03 18:59:05.994397+00:00
[2024-11-03T18:59:28.179+0000] {standard_task_runner.py:55} INFO - Started process 643 to run task
[2024-11-03T18:59:28.184+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_Data_Preprocessing', 'encode_data_task', 'manual__2024-11-03T18:59:05.994397+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmphmp82zra']
[2024-11-03T18:59:28.187+0000] {standard_task_runner.py:83} INFO - Job 50: Subtask encode_data_task
[2024-11-03T18:59:28.303+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_Data_Preprocessing.encode_data_task manual__2024-11-03T18:59:05.994397+00:00 [running]> on host a95df74735a7
[2024-11-03T18:59:28.441+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_Data_Preprocessing
AIRFLOW_CTX_TASK_ID=encode_data_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-03T18:59:05.994397+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-03T18:59:05.994397+00:00
[2024-11-03T18:59:28.460+0000] {airflowdag.py:103} ERROR - No data found in XCom for key 'cleaned_data'
[2024-11-03T18:59:28.460+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-11-03T18:59:28.476+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=DAG_Data_Preprocessing, task_id=encode_data_task, execution_date=20241103T185905, start_date=20241103T185928, end_date=20241103T185928
[2024-11-03T18:59:28.528+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-11-03T18:59:28.562+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
